### ISSUE TITLE
Atmospherics should have Firelock electronics

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/yogstation-classic#256]
Just like Engineering has airlock electronics in tool vendor. This way we wouldn't need to run to cargo every time one is destroyed..

**Comment by Super3222** [comment]
Eh... Engineers are usually the ones who work with machinery / doors and Atmospheric Technicians are the ones who work with getting air in or out of places...I'm not sure about this.

**Comment by 00aquir** [comment]
My main argument is that while they could, engineers usually care even less about firelocks and are happy if they can deal with several broken airlocks at the same time (at a big explosion) + that Atmos techs would be able to contribute more (and not just with pipes, really - engineers could basically repair anything without Atmos techs if they had the RPD..), and while Atmos techs have access to change access permissions on airlock electronics, firelocks don't even need that + are not hackable so they can't shock you.

Basically what I'm asking is that at least let atmos have some firelock boards on the atmos table (like the airlock boards on the engineering table) for the firefighting department.

**Comment by DavidEdwards** [comment]
I think this one is do-able. I can't see any balance issues. Leaving this one open to the newer contributors.

**Comment by Cruix** [comment]
Is there some way to make the small firelocks in-game? Whenever I mess with one, it turns into a full-tile firelock.

**Comment by Paroclomas** [comment]
There are some in aux tool storage, i think that takes care of it, taking into consideration that atmos shouldnt be fixing anything else than pipes
